### DISCUSSION TITLE
Conditoinal rendering of character based on character width

### DISCUSSION THREAD

**Discussion by nibirrayy** [starship/starship#2868] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/12048453/124803603-f1c0a000-df76-11eb-91bd-a5c8aec8884b.png)

i am trying to configure starship in way so that I can split the output in two different sections, one on the left and one on the right. the problem is I am using a config like this

```
format = """
$shell
$directory\
$username\
$battery\
$custom ─────────────────────────────────────── $git_status
$character
"""
```

this is not ideal cause we are using using a dash to push $git_status to side and if the terminal font is too large it is wrapping to the next line 
something like this
![image](https://user-images.githubusercontent.com/12048453/124804117-9216c480-df77-11eb-8c93-f679514ec013.png)

is there is any way to make this conditional were we check character width and then set the number of character or something

**Comment by matchai** [comment, marked as answer]
This is currently an unsupported feature in Starship.
Please follow this issue for updates: https://github.com/starship/starship/issues/498
